### ISSUE TITLE
Json files making no difference to spawning.

### ISSUE THREAD

**Issue by JustLiam040** [McJty/InControl#64]
The issue: Changing the json file is having no affect on mob spawning whatsoever.
What i expected to happen; i expected parrots to be able to spawn in this BOP biome (Xeric Shrubland).
My version of minecraft: 1.12.2

What happened: Parrots were not spawning in the biome at all.

Description. I'm making a modpack with custom world gen, there is no jungle and thus no way to get parrots so i figured this was a great way to sort this issue, expecting to be able to change parrots to spawn in The Xeric Shrubland biome, as this is one of the few biomes im using in the world gen. I tried using the biome variable rather than the block to get it to spawn. I tried the numerical biome id (94) and what i would imagine is the written id (xeric_shrubland). Neither made any difference. Attached is a snipping tool of the code. Which youll notice is basically just a rewritten version of the example code.
![error](https://user-images.githubusercontent.com/33065753/41412542-492597c2-6fd8-11e8-8950-4c7b35340d0f.PNG)

Any help would be appreciated as id really like to get this working. I tried the /ctrlloadpotentialspawn command in game and was just met with an error message that reads "An unknown error occured when attempting to perform this command".

Im not sure whether this is an error with my code, or whether InControl is incompatible with biomes o'plenty.

I installed biome tweaker to see whether that would give me the ID or not, i tried its resource location which was "biomesoplenty:xeric_shrubland" and also tried the numeric ID that it gave me. I was still getting no parrots and the unknown error. 

I also tried changing the variable from "biome" to "biometype" so that i could set it to hot as that's the climate of the biome, still no luck unfortunately :(

Thanks for any help you provide.

**Comment by JustLiam040** [comment]
Oh one last thing, i installed biome tweaker to see whether that would give me the ID or not, i tried its resource location which was "biomesoplenty:xeric_shrubland" and also tried the numeric ID that it gave me. I was still getting no parrots and the unknown error.

**Comment by McJty** [comment]
Don't use numeric biome id's. Use the proper ID in the format <modid>:<name>
You also might want to use 'minecraft:parrot' instead of 'Parrot'.

Restart the game after making changes if the command doesn't work

**Comment by JustLiam040** [comment]
Hi MCJ, I'll definitely try the minecraft:parrot entity id rather than simply Parrot. However, I did try with multiple ID's and none had any success. Not only did i try the numeric id i also tried the resource id, the regular id and tried switching to biome type to use the "HOT" dictionary rather than simply the biome.

I will try the new entity ID when im next able to, if this is unsuccessful i will comment on the post with a photo of my new code and try to explain my thought process behind it all. 

Thank you for the quick reply.

**Comment by JustLiam040** [comment]
Hi MCJ, thanks for your quick reply. I tried the suggestions youve made yet im still getting no results. The code will be shown in the image below. I have tried the entity tag "minecraft:parrot", Parrot, parrot and minecraft:Parrot yet none of them seem to spawn. I'm really unsure of what this is, and to be honest i really dont understand what im doing wrong. 

The new code;

![capture](https://user-images.githubusercontent.com/33065753/41441352-28f37b02-702a-11e8-94be-3d5d85895241.PNG)

This is one of the first mobs i want to try and change the spawn of, i will change more later however cannot get onto those until i manage to get this one working.
